internal/core/src/mmap/Column.h

```diff
@@ -115,6 +132,11 @@ class ColumnBase {
         return num_rows_;
     };
 
+    const size_t
+    ByteSize() const {
```

Comment:
Size() ？ @yah01

Reply:
it's not the same, `Size()` returns the length of column (like `std::vector::size()`), and `ByteSize()` returns the size of the column in bytes